ecologits/impacts/llm.py

```diff
@@ -150,7 +207,9 @@ def server_energy(
 
 @dag.asset
 def request_energy(
-        datacenter_pue: float,
+        provider: str,
+        provider_pue: dict,
+        ai_company_to_data_center_provider: dict,
```

Comment:
As mentioned above, keep this simple; only use the `datacenter_pue` as input. This value of PUE will be retrieved before this function call with the right provider.